tests/integration/security/fuzz/fuzz_test.go

```diff
@@ -45,44 +49,138 @@ spec:
 `
 )
 
+var (
+	// Known unsupported path parameter ("/bla;foo") normalization for Tomcat.
+	dotdotPwnIgnoreTomcat = []string{
+		"/../private/secret.html;index.html <- VULNERABLE!",
+		"/../private/secret.html;index.htm <- VULNERABLE!",
+		"/..%5Cprivate%5Csecret.html;index.html <- VULNERABLE!",
+		"/..%5Cprivate%5Csecret.html;index.htm <- VULNERABLE!",
+	}
+	wfuzzIgnoreTomcat = []string{
```

Comment:
can you explain this?

Reply:
This is actually the same as the `dotdotPwnIgnoreTomcat` for the path parameter (e.g. "bla;foo")

Reply:
I don't understand how we just have this list? Aren't their infinite paths that will normalize to the same URL that would have to be here?

Reply:
the fuzzer we used only generates finite number of paths in the test, that's why we can just ignore these when testing with the tomcat server.